### PR TITLE
(16) feat: update localization

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- update localization for ru-ru, da and pt-br
- translate the terms for pt-br

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: locale/pt-br/texts.po [hunk 1]
```diff
@@ -935,49 +1113,55 @@ msgstr "A transação ainda não chegou na sua carteira. Deseja tentar novamente
...
-msgstr "Ao clicar em aprovar, você assinará a mensagem solicitada usando o primeiro "
-"endereço derivado da sua chave raiz no caminho de derivação m/44'/280'/0'/0/0."
+msgstr ""
+"Ao clicar em aprovar, você assinará a mensagem solicitada usando o primeiro "
+"endereço derivado da sua chave raiz no caminho de derivação "
+"m/44'/280'/0'/0/0."
+
+#: src/components/NetworkSettings/NetworkStatusBar.js:14
+msgid "Custom network"
+msgstr ""
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:1164]: Missing translation

File: locale/pt-br/texts.po [hunk 2]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+#: src/screens/UnregisterToken.js:139
 msgid "Unregister token"
 msgstr "Desregistrar token"
 
-#: src/sagas/pushNotification.js:56
+#: src/screens/WalletConnect/WalletConnectList.js:33
+#, fuzzy
+msgid "There was an error connecting. Please try again later."
+msgstr ""
+"Ocorreu um erro ao atualizar as configurações de notificação. Por favor, "
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:802]: Is this correct?

File: locale/pt-br/texts.po [hunk 3]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+"tente novamente mais tarde."
+
+#. translator: Used when the QR Code Scanner is opened, and user will manually
+#. enter the information.
+#: src/screens/WalletConnect/WalletConnectList.js:111
+#, fuzzy
+msgid "Add"
+msgstr "Endereço"
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:810]: Is this correct?

File: locale/pt-br/texts.po [hunk 4]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+
+#: src/screens/WalletConnect/WalletConnectList.js:117
+msgid "End session"
+msgstr ""
+
+#: src/screens/WalletConnect/WalletConnectList.js:117
+#, fuzzy
+msgid "This will disconnect the session."
+msgstr "Esta wallet está conectada à **mainnet**."
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:819]: Is this correct?

File: locale/pt-br/texts.po [hunk 5]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+#: src/screens/NetworkSettings/CustomNetworkSettingsScreen.js:20
+msgid "Custom Network Settings"
+msgstr "Configurações de Rede Personalizada"
+
+#: src/screens/NetworkSettings/CustomNetworkSettingsScreen.js:21
+msgid ""
+"Any token outside mainnet network bear no value. Only change if you know "
+"what you are doing."
+msgstr ""
+"Nenhum token fora da rede mainnet possui valor. Apenas mude a rede ser você "
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:860]:
```suggestion
"Nenhum token fora da rede mainnet possui valor. Apenas mude a rede se você "
```

File: locale/pt-br/texts.po [hunk 6]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+msgid "Risk Disclaimer"
+msgstr "Aviso de Risco"
+
+#: src/screens/NetworkSettings/NetworkSettingsDisclaimerScreen.js:20
+msgid ""
+"Do not change this unless you know what you are doing, and under no "
+"circumstances change these settings based on someone else suggestion, as "
+"this can potentially make you susceptible to fraudulent schemes."
+msgstr ""
+"Não altere a rede a menos que você saiba o que está fazendo, e sub nenhuma "
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:932]:
```suggestion
"Não altere a rede a menos que você saiba o que está fazendo, e sob nenhuma "
```

**Review by andreabadesso — APPROVED** [review]

File: locale/pt-br/texts.po [hunk 7]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+#. translator: Used when the QR Code Scanner is opened, and user will manually
+#. enter the information.
+#: src/screens/WalletConnect/WalletConnectList.js:111
+#, fuzzy
+msgid "Add"
+msgstr "Adicionar"
+
+#: src/screens/WalletConnect/WalletConnectList.js:117
+msgid "End session"
+msgstr ""
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:814]:
```suggestion
msgstr "Encerrar Sessão"
```

File: locale/pt-br/texts.po [hunk 8]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+
+#: src/screens/WalletConnect/WalletConnectList.js:117
+#, fuzzy
+msgid "This will disconnect the session."
+msgstr "Esta wallet está conectada na sessão."
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:819]:
```suggestion
msgstr "Isso irá desconectar a sessão."
```

File: locale/pt-br/texts.po [hunk 9]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+
+#: src/screens/WalletConnect/WalletConnectList.js:119
+msgid "End"
+msgstr ""
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:823]:
```suggestion
msgstr "Encerrar"
```

File: locale/pt-br/texts.po [hunk 10]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+
+#: src/screens/WalletConnect/WalletConnectList.js:137
+msgid "Wallet Connect Sessions"
+msgstr ""
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:827]:
```suggestion
msgstr "Sessões do Wallet Connect"
```

File: locale/pt-br/texts.po [hunk 11]
```diff
@@ -698,92 +700,268 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+
+#: src/screens/WalletConnect/WalletConnectManual.js:45
+msgid "Wallet Connect URI"
+msgstr ""
+
+#: src/screens/WalletConnect/WalletConnectManual.js:57
+#: src/screens/WalletConnect/WalletConnectScan.js:34
+#, fuzzy
+msgid "Connect"
+msgstr "Conectado ao servidor"
```
Comment (pedroferreira1) [inline locale/pt-br/texts.po:842]:
```suggestion
msgstr "Conectar"
```

Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Applied at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/1d122b2b7dce88bbed1f22bc6e1722a41178e365

File: locale/pt-br/texts.po (hunk 10 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Applied at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/1d122b2b7dce88bbed1f22bc6e1722a41178e365

File: locale/pt-br/texts.po (hunk 9 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Applied at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/1d122b2b7dce88bbed1f22bc6e1722a41178e365

File: locale/pt-br/texts.po (hunk 8 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Applied at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/1d122b2b7dce88bbed1f22bc6e1722a41178e365

File: locale/pt-br/texts.po (hunk 7 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Applied at: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/1d122b2b7dce88bbed1f22bc6e1722a41178e365

**Review by pedroferreira1 — APPROVED** [review]

File: locale/pt-br/texts.po [hunk 12]
```diff
@@ -698,92 +700,267 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+msgid "Wallet Connect Sessions"
+msgstr "Sessões do Wallet Connect"
+
+#: src/screens/WalletConnect/WalletConnectManual.js:39
+msgid "Manual Connection"
+msgstr "Digitar"
+
+#: src/screens/WalletConnect/WalletConnectManual.js:45
+msgid "Wallet Connect URI"
+msgstr ""
```
Comment (andreabadesso) [inline locale/pt-br/texts.po:831]:
```suggestion
msgstr "URI do Wallet Connect"
```

Sounds weird in portuguese?

File: locale/pt-br/texts.po [hunk 13]
```diff
@@ -698,92 +700,267 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+"Any token outside mainnet network bear no value. Only change if you know "
+"what you are doing."
+msgstr ""
+"Nenhum token fora da rede mainnet possui valor. Apenas mude a rede se você "
+"souber o que está fazendo."
+
+#: src/screens/NetworkSettings/CustomNetworkSettingsScreen.js:18
+#: src/screens/NetworkSettings/helper.js:4
+msgid "Updating custom network settings..."
+msgstr "Atualizando configurações de rede…"
```
Comment (andreabadesso) [inline locale/pt-br/texts.po:859]:
```suggestion
msgstr "Atualizando configurações de rede..."
```

Comment (andreabadesso) [reply locale/pt-br/texts.po:859]: Was that a symbol for "..."? Was it on purpose?

File: locale/pt-br/texts.po [hunk 14]
```diff
@@ -698,92 +700,267 @@ msgstr "Sua transferência de **${ this.amountAndToken }** foi confirmada"
...
+"circumstances change these settings based on someone else suggestion, as "
+"this can potentially make you susceptible to fraudulent schemes."
+msgstr ""
+"Não altere a rede a menos que você saiba o que está fazendo, e sob nenhuma "
+"circunstância altere a rede por sugestão de terceiros, uma vez que ao "
+"alterar a rede você pode ser vítima de fraude."
+
+#: src/sagas/networkSettings.js:81
+msgid "Custom Network Settings cannot be empty."
+msgstr "Configurações de Rede não pode estar vazia."
```
Comment (andreabadesso) [inline locale/pt-br/texts.po:931]:
```suggestion
msgstr "As Configurações de Rede não podem estar vazias."
```

**Review by andreabadesso — CHANGES_REQUESTED** [review]

File: locale/pt-br/texts.po (hunk 13 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po:859]: No. The localization tool should have captured my ligature glyph, somehow.

File: locale/pt-br/texts.po (hunk 14 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Fixed: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/3930be656af6644b3c011bb00a2f51358eb2e5f5

File: locale/pt-br/texts.po (hunk 13 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po]: Fixed: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/361/commits/3930be656af6644b3c011bb00a2f51358eb2e5f5

File: locale/pt-br/texts.po (hunk 12 above)
Comment (alexruzenhack) [reply locale/pt-br/texts.po:831]: I will let it without translation.

Comment (andreabadesso) [reply locale/pt-br/texts.po:831]: Agreed

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**